web/src/main/resources/application.properties

```diff
@@ -367,4 +367,15 @@ kundportalen.service.codes.eleg={'Webcert e-leg'}
 kundportalen.service.codes.siths={'Webcert integrerad-direktanslutning','Webcert som Agent','Webcert','Webcert SITHS','Webcertintegrerad','Intygstj�nster'}
 kundportalen.connection.request.timeout=5000
 kundportalen.connection.timeout=5000
-kundportalen.read.timeout=5000
\ No newline at end of file
+kundportalen.read.timeout=5000
+
+servicenow.base.url=
```

Comment:
Should this be removed so it forces us to set these config for each environment? Same for the username and password.

Reply:
done